packages/common/test/search/_internal/portalSearchGroupMembers.test.ts

```diff
@@ -0,0 +1,335 @@
+      )[1] as Portal.ISearchGroupUsersOptions;
+      expect(callOpts.memberType).toBe("admin");
+      expect(callOpts.name).toBe("steve");
+      expect(response.results.length).toBe(2);
+      const user1 = response.results[0];
+      expect(user1.owner).toBe("e2e_pre_hub_c_member");
+      expect(user1.orgId).not.toBeDefined();
+      expect(user1.orgName).not.toBeDefined();
+      expect(user1.memberType).toBe("admin");
+      expect(user1.family).toBe("people");
+    });
+    it("search without auth", async () => {
+      const searchSpy = spyOn(Portal, "searchGroupUsers").and.callFake(() => {
+        return Promise.resolve(cloneObject(SparseGroupMembersResponse));
+      });
+      const userSpy = spyOn(Portal, "getUser").and.callFake((options: any) => {
+        const resp = cloneObject(SparseUser);
+        resp.username = options.username;
+        return Promise.resolve(resp);
+      });
+      // NOTE: enrichUserSearchResult is tested elsewhere so we don't assert on the results here
+      const enrichUserSearchResultSpy = spyOn(
+        users,
+        "enrichUserSearchResult"
+      ).and.callThrough();
+      const opts: IHubSearchOptions = {
+        num: 2,
+        requestOptions: {
+          portal: "https://www.arcgis.com/sharing/rest",
+        },
+      };
+      const response = await portalSearchGroupMembers(qry, opts);
+      // validate spy calls
+      expect(searchSpy.calls.count()).toBe(1, "should call searchGroupUsers");
+      expect(userSpy.calls.count()).toBe(2, "should get each user");
+      expect(enrichUserSearchResultSpy.calls.count()).toBe(
+        2,
+        "should enrich each user"
+      );
+      // validate call args
+      const groupId = searchSpy.calls.argsFor(0)[0];
+      expect(groupId).toBe("3ef");
+      const callOpts = searchSpy.calls.argsFor(
+        0
+      )[1] as Portal.ISearchGroupUsersOptions;
+      expect(callOpts.memberType).toBe("admin");
+      expect(callOpts.name).toBe("steve");
+      expect(response.results.length).toBe(2);
+      const user1 = response.results[0];
+
+      expect(user1.owner).toBe("e2e_pre_hub_c_member");
+      expect(user1.orgId).not.toBeDefined();
+      expect(user1.orgName).not.toBeDefined();
+      expect(user1.memberType).toBe("admin");
+      expect(user1.family).toBe("people");
+    });
+  });
+});
+
+const SparseUser: Portal.IUser = {
+  username: "e2e_pre_hub_c_member",
+  udn: null,
+  id: "726f96ef603b4551a1fb76fa99f68364",
+  fullName: "ArcGIS HubE2E",
+  firstName: "ArcGIS",
+  lastName: "HubE2E",
+  description: null,
+  tags: ["org:Q05waIVQNYv92Kgz", "hubRole:participant"],
+  culture: "",
+  cultureFormat: "",
+  region: "US",
+  units: "english",
+  thumbnail: null,
+  access: "public",
+  created: 1569441945000,
+  modified: 1569442018000,
+  provider: "arcgis",
+} as unknown as Portal.IUser;
+
+const GroupMembersResponse = {
+  total: 2,
+  start: 1,
+  num: 25,
+  nextStart: -1,
+  owner: {
+    username: "e2e_pre_pub_admin",
```

Comment:
Isn't this an actual user? If it is, we should change the example since hub.js is open source